crates/release_plz_core/src/git/backend.rs

```diff
@@ -540,23 +555,37 @@ impl GitClient {
         };
 
         info!("opened pr: {}", git_pr.html_url);
-        self.add_labels(pr, git_pr.number)
-            .await
-            .context("Failed to add labels")?;
+        if pr.labels.is_empty() {
+            warn!("No labels provided for PR #{}", git_pr.number);
```

Comment:
why emitting a warning if no pr labels are provided?
I think it's just noise for people who don't use pr labels